applications/python-the-coffee-bar-apps/src/bar/coffee_machine.py

```diff
@@ -34,7 +36,7 @@ def __init__(self, name: str = 'The Coffee Machine', host: str = 'localhost', po
         self.add_all_endpoints()
 
         # Increase CPU usage for some time and add network delay
-        self.scheduler = BackgroundScheduler()
+        self.scheduler = BackgroundScheduler(job_defaults={'max_instances': 2})
```

Comment:
Why do we need to set `max_instances` to `2`?

Reply:
this was added to avoid intermittent errors message saying that no scheduler instances are available